tools/mo/setup.py

```diff
@@ -109,7 +109,7 @@ def find_package_modules(self, package, package_dir):
       'mxnet': read_text('requirements_mxnet.txt'),
       'onnx': read_text('requirements_onnx.txt'),
       'tensorflow': read_text('requirements_tf.txt'),
-      'tensorflow2': read_text('requirements_tf2.txt'),
+      'tensorflow2': read_text('requirements_tf.txt'),
```

Comment:
In general I don't understand why the file is renamed. Deprecation does not mean that user cannot use tf, it means that such support will be dropped in some time

Reply:
See comment above from Andrey Kochin.